woodwork/type_sys/utils.py

```diff
@@ -11,10 +11,14 @@
 
 def col_is_datetime(col, datetime_format=None):
     """Determine if a dataframe column contains datetime values or not. Returns True if column
-    contains datetimes, False if not. Optionally specify the datetime format string for the column."""
+    contains datetimes, False if not. Optionally specify the datetime format string for the column.
+    Will not infer float/double data as datetime."""
```

Comment:
Might want to mention integers here as well since this is being added. As I understand it, this will also not identify integer columns as valid datetime columns.

Reply:
Integers will be unchanged here! Just floats will be not converted to datetime.

Reply:
Yes, but we are now specifically calling out that float/double will not be inferred as a datetime. This is also true for integers, but that isn't mentioned. For consistency if we mention the behavior for float/double we should also mention it for integers.

Reply:
Currently, the check that I added was to return `false` if the column is a float dtype. I don't add anything here to do the same for integer, and if an integer can be interpreted as a datetime (through string conversion), we will return true. You can look at the new test I added!

Reply:
After discussion with @thehomebrewnerd, we decided that all numeric dtypes should be left as numerics rather than inferred as datetimes.